lineapy/graph_reader/session_artifacts.py

```diff
@@ -42,8 +45,8 @@ class SessionArtifacts:
     _session_id: LineaID
     graph: Graph
     db: RelationalLineaDB
-    artifact_nodecollections: List[NodeCollection]
-    import_nodecollection: NodeCollection
+    artifact_nodecollections: List[UserCodeNodeCollection]
```

Comment:
Maybe we want to rename this variable name to something like `usercode_nodecollections` to be consistent with what types of `NodeCollection` it holds.

Reply:
will do, I agree